package.json

```diff
@@ -57,81 +32,21 @@
         "gatsby": "^2.19.27",
         "glob": "^7.1.3",
         "husky": "^2.5.0",
-        "jest": "^25.1.0",
-        "jest-expect-message": "^1.0.2",
-        "jest-junit": "^6.4.0",
+        "lerna": "^4.0.0",
         "lint-staged": "^8.2.1",
         "node-fetch": "^2.3.0",
         "oberon-prettier-config": "^1.0.0",
         "prettier": "^1.19.1",
-        "snyk": "^1.361.2",
+        "serverless": "^1.35.1",
+        "serverless-s3-sync": "^1.7.1",
         "tslint": "^5.18.0",
         "tslint-config-oberon": "^1.1.1",
```

Comment:
tslint is dead, replaced by eslint.
these are removable: https://www.npmjs.com/package/tslint

Reply:
Good call. We probably should've removed it back in #124.